src/background/background-process.js

```diff
@@ -268,10 +269,10 @@ export default class BackgroundProcess {
    * @return {Promise} Will be resolved when all scripts are loaded successfully.
    */
   loadContentData (tabScript) {
-    let polyfillScript = this.loadPolyfill(tabScript.tabObj.tabId)
-    let contentScript = this.loadContentScript(tabScript.tabObj.tabId)
-    let contentCSS = []
-    for (let fileName of this.settings.contentCSSFileNames) { // eslint-disable-line no-unused-vars
+    const polyfillScript = this.loadPolyfill(tabScript.tabObj.tabId)
+    const contentScript = this.loadContentScript(tabScript.tabObj.tabId)
+    const contentCSS = []
```

Comment:
We changing this variable - than `const` is not appropriate here.

Reply:
ok, will override the linter